src/woo_publications/config/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+from django.utils.encoding import force_str
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+from zgw_consumers.constants import APITypes
+
+from .validators import validate_rsin
+
+
+class GlobalConfiguration(SingletonModel):
+    """
+    Global configuration settings managed via the admin.
+
+    .. note:: This models a shortcut where we currently only support a single
+       Documents API. Our database can handle multiple, but a mechanism to specify/route
+       this is currently out of scope.
+    """
+
+    documents_api_service = models.ForeignKey(
+        "zgw_consumers.Service",
+        on_delete=models.PROTECT,
+        limit_choices_to={"api_type": APITypes.drc},
+        verbose_name=_("Documents API service"),
+        help_text=_(
+            "The service to use for new document uploads - the metadata and binary "
+            "content will be sent to this API."
+        ),
+        null=True,
+        blank=False,
```

Comment:
What is the reason why these two are different?

Reply:
it's a required field, but django-solo creates the record if it doesn't exist yet, and making it not-nullable crashes with integrity error as there may not be any services defined yet while this record is being created.

`blank=False` ensures that form validation in the admin prevents you from leaving it empty

this pattern is one of the reasons I'm not too fond of django-solo anymore